app/src/main/java/com/spencerpages/collections/ProofSets.java

```diff
@@ -0,0 +1,87 @@
+package com.spencerpages.collections;
+
+import android.database.sqlite.SQLiteDatabase;
+
+import com.coincollection.CoinPageCreator;
+import com.coincollection.CoinSlot;
+import com.coincollection.CollectionInfo;
+import com.coincollection.CollectionListInfo;
+import com.spencerpages.R;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class ProofSets extends CollectionInfo {
+    public static final String COLLECTION_TYPE = "ProofSets";
+
+    private static final int REVERSE_IMAGE = R.drawable.a24rg;
+
+    private static final int OBVERSE_IMAGE_COLLECTED = R.drawable.a24rg;
+
+    @Override
+    public String getCoinType() {return COLLECTION_TYPE;}
+
+    @Override
+    public int getCoinImageIdentifier() {return REVERSE_IMAGE;}
+
+    private static final Integer START_YEAR = 1950;
+    private static final Integer STOP_YEAR = 2024;
```

Comment:
Set this to STILL_IN_PRODUCTION so that it updates automatically every year
```suggestion
    private static final Integer STOP_YEAR = CoinPageCreator.OPTVAL_STILL_IN_PRODUCTION;
```